# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/cross-origin-opener-policy/navigate-top-to-aboutblank.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/html/cross-origin-opener-policy/navigate-top-to-aboutblank.https.html.ini
--- a/testing/web-platform/meta/html/cross-origin-opener-policy/navigate-top-to-aboutblank.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/html/cross-origin-opener-policy/navigate-top-to-aboutblank.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,14 +1,15 @@
 [navigate-top-to-aboutblank.https.html]
   expected:
-    if (os == "win") and not webrender and debug and (processor == "x86_64"): ["OK", "TIMEOUT", "CRASH"]
-    if (os == "win") and not webrender and debug and (processor == "x86"): OK
-    if (os == "win") and webrender and debug and not swgl: ["OK", "TIMEOUT", "CRASH"]
-    if (os == "win") and webrender and debug and swgl: OK
-    if (os == "win") and not webrender and not debug: ["OK", "TIMEOUT", "CRASH"]
-    if (os == "win") and webrender and not debug: OK
-    if (os == "linux") and webrender and not debug: OK
+    if (os == "win") and debug and (processor == "x86_64") and webrender and not swgl: ["CRASH", "OK", "TIMEOUT"]
+    if (os == "win") and debug and (processor == "x86_64") and webrender and swgl: ["OK", "TIMEOUT", "CRASH"]
+    if (os == "win") and debug and (processor == "x86_64") and not webrender: ["OK", "TIMEOUT", "CRASH"]
+    if (os == "win") and not debug and webrender and fission: OK
+    if (os == "win") and not debug and webrender and not fission: ["OK", "TIMEOUT"]
+    if (os == "linux") and webrender and not debug and fission: OK
+    if (os == "win") and debug and (processor == "x86"): ["OK", "TIMEOUT"]
+    if (os == "win") and not debug and not webrender: ["OK", "TIMEOUT", "CRASH"]
+    if (os == "mac") and not debug: ["OK", "TIMEOUT", "CRASH"]
     if (os == "mac") and debug: OK
-    if (os == "mac") and not debug: ["OK", "TIMEOUT", "CRASH"]
     [OK, CRASH]
   [Navigate top to about:blank from iframe with opener COOP: |header(Cross-Origin-Opener-Policy,same-origin), iframe origin: https://www1.web-platform.test:8443]
     expected: FAIL